xml/obs_best_practice_manage_group.xml

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE sect1
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<sect1 version="5.1" xml:id="manage_group"
+ xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" >
+  <title>Manage Group</title>
+  <para>
+    Only administrators and users with the role Maintainer can manage groups.
+    They can add and remove other users from the group, as well as give them
+    Maintainer's rights.
+  </para>
+  <para>
+    On the Group Members tab there is a link to Add User, then enter the name
```

Comment:
On the group Members tab<b>, (missing comma)</b>, there is a link to Add **Member**